includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -279,4 +288,22 @@ public static function render_persist_inbox_field() {
 		</p>
 		<?php
 	}
+
+	/**
+	 * Render object type field.
+	 */
+	public static function render_object_type_field() {
+		$value = \get_option( 'activitypub_object_type', ACTIVITYPUB_DEFAULT_OBJECT_TYPE );
+		?>
+		<p>
+			<label>
+				<input type="checkbox" name="activitypub_object_type" value="note" <?php \checked( 'note', $value ); ?> />
+				<?php \esc_html_e( 'Use Template Tags instead of letting the plugin choose the best possible format for you.', 'activitypub' ); ?>
+			</label>
+		</p>
+		<p class="description">
+			<?php \esc_html_e( 'This is mainly for backwards compatibility. It is not recommended to use the Template Tags, because it might not be supported in future versions.', 'activitypub' ); ?>
```

Comment:
```suggestion
			<?php \esc_html_e( 'This is mainly for backwards compatibility. It is not recommended to use the Template Tags, because they will be removed in a future version.', 'activitypub' ); ?>
```
Optional: I think it's fine to be more definitive here.